Detections/SecurityEvent/UserCreatedAddedToBuiltinAdmins_1d.yaml

```diff
@@ -22,21 +25,51 @@ relevantTechniques:
   - T1098
   - T1078
 query: |
-  SecurityEvent
+  (union isfuzzy=true
+  (SecurityEvent
   | where EventID == 4720
   | where AccountType == "User"
   | project CreatedUserTime = TimeGenerated, CreatedUserEventID = EventID, CreatedUserActivity = Activity, Computer = toupper(Computer), 
   CreatedUser = tolower(TargetAccount), CreatedUserSid = TargetSid, AccountUsedToCreateUser = strcat(SubjectAccount), SidofAccountUsedToCreateUser = SubjectUserSid
-  | join (
-  SecurityEvent 
+  ),
+  (WindowsEvent
+  | where EventID == 4720 and EventData has "User"
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend SubjectAccount = strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(SubjectAccount endswith "$" or SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(SubjectUserSid), "", "User")
+  | where AccountType == "User"
+  | extend TargetAccount = strcat(EventData.TargetDomainName,"\\", EventData.TargetUserName)
+  | extend Activity="4720 - A user account was created."
+  | extend TargetSid = tostring(EventData.TargetSid)
+  | project CreatedUserTime = TimeGenerated, CreatedUserEventID = EventID, CreatedUserActivity = Activity, Computer = toupper(Computer), 
+  CreatedUser = tolower(TargetAccount), CreatedUserSid = TargetSid, AccountUsedToCreateUser = strcat(SubjectAccount), SidofAccountUsedToCreateUser = SubjectUserSid
+  ))
+  | join ((union isfuzzy=true
+  (SecurityEvent 
   | where AccountType == "User"
   // 4732 - A member was added to a security-enabled local group
   | where EventID == 4732
-  //TargetSid is the builin Admins group: S-1-5-32-544
+  // TargetSid is the builin Admins group: S-1-5-32-544
   | where TargetSid == "S-1-5-32-544"
   | project GroupAddTime = TimeGenerated, GroupAddEventID = EventID, GroupAddActivity = Activity, Computer = toupper(Computer), GroupName = tolower(TargetAccount), 
   GroupSid = TargetSid, AccountThatAddedUser = SubjectAccount, SIDofAccountThatAddedUser = SubjectUserSid, CreatedUserSid = MemberSid
-  )
+  ),
+  (  WindowsEvent 
+  // 4732 - A member was added to a security-enabled local group
+  | where EventID == 4732 and EventData has "User" and EventData has "S-1-5-32-544"
+  //TargetSid is the builin Admins group: S-1-5-32-544
+  | extend SubjectUserSid = tostring(EventData.SubjectUserSid)
+  | extend SubjectAccount = strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend AccountType=case(SubjectAccount endswith "$" or SubjectUserSid in ("S-1-5-18", "S-1-5-19", "S-1-5-20"), "Machine", isempty(SubjectUserSid), "", "User")
+  | where AccountType == "User"
+  | extend TargetAccount = strcat(EventData.TargetDomainName,"\\", EventData.TargetUserName)
+  | extend TargetSid = tostring(EventData.TargetSid)
```

Comment:
Not criticial, but for the record: line 65 should be after 66/67 - conditions first!